example/rest.py

```diff
@@ -55,4 +74,61 @@ def main():
     # print(profile['payload']['username'])
 
 
-main()
+# main()
+
+# Deposits
+def ethereumDeposit():
+    try:
+        client = Client('testnet')
+        try:
+            # login to use private endpoints
+            login = client.complete_login(ETH_ADDRESS, PRIVATE_KEY)
+            print('Login Successful')
+        # Error: AuthenticationError | requests.exceptions.HTTPError
+        except exception.AuthenticationError as exc:
+            print(exc)
+        except requests.exceptions.HTTPError as exc:
+            print(exc.response.json())
+
+        user_signature = create_user_signature(PRIVATE_KEY, 'testnet')
+        key_pair = get_stark_key_pair_from_signature(user_signature)
+        stark_public_key = key_pair['stark_public_key']
+        provider = Web3(Web3.HTTPProvider(rpc_provider))
+        signer = Account.from_key(PRIVATE_KEY)
+
+        # # deposit with eth private key
+        # deposit_res_with_private_key = client.deposit_from_ethereum_network(
```

Comment:
Running this function gives this error (moved outside try except block to get traceback)
```
<class 'eth_account.signers.local.LocalAccount'>
Traceback (most recent call last):
  File "example/rest.py", line 134, in <module>
    ethereumDeposit()
  File "example/rest.py", line 125, in ethereumDeposit
    deposit_res_with_private_key = client.deposit_from_ethereum_network(
  File "/home/austin/Documents/brine/tanx-connector-python/src/brineconnector/client.py", line 345, in deposit_from_ethereum_network
    return self.deposit_from_ethereum_network_with_starkKey(signer, provider, f'0x{stark_public_key}', amount, currency)
  File "/home/austin/Documents/brine/tanx-connector-python/src/brineconnector/client.py", line 273, in deposit_from_ethereum_network_with_starkKey
    parsed_amount = w3.toWei(amount, 'ether')
AttributeError: 'Web3' object has no attribute 'toWei'
```